environments/all/Brewfile

```diff
@@ -83,10 +87,12 @@ cask 'quicklook-csv'   # preview csvs
 cask 'qlvideo'         # preview more video types
 
 # Install App Store packages
-mas 'Keynote', id: 409_183_694
+# for som reason, Keynote, Numbers, and Pages insist on being reinstalled on
+# everr run, so, for the time being, they've been disabled.
```

Comment:
```suggestion
# for some reason, Keynote, Numbers, and Pages insist on being reinstalled on
# every run, so, for the time being, they've been disabled.
```

Reply:
Hm, darn, the suggestion didn't take for multiple lines.

Reply:
were you just trying to correct "everr" to "every"? or was there more wrong there?

Reply:
Also “for som reason”

Reply:
Fixed